serverless/serverless-experimental.yml

```diff
@@ -48,6 +48,9 @@ functions:
   reg_core_migrate:
     handler: migrate.handler
     timeout: 15
+  create_su:
```

Comment:
Need to add the env vars for the username / password. Can probably reuse the same ssm vars used for the `HTTP_AUTH` env vars `${ssm:/eregulatinos/http/user~true}` `${ssm:/eregulations/http/password~true}`